docker/docker-compose.yml

```diff
@@ -62,14 +37,9 @@ services:
       - MYSQL_ROOT_PASSWORD=root
       - MYSQL_DATABASE=tests
       - MYSQL_USER=prisma
-      - MYSQL_PASSWORD=prisma
-    # https://stackoverflow.com/questions/55559386/how-to-fix-mbind-operation-not-permitted-in-mysql-error-log
-    cap_add:
-      - SYS_NICE # CAP_SYS_NICE
-    volumes:
-      - mysql:/var/lib/mysql
```

Comment:
This seems unrelated and was probably needed for some reason, why removing it now?

Reply:
I aligned our config on the one from the engines. Volumes aren't needed and this error does not happen anymore.